### ISSUE TITLE
Doc clarification around updateOne/updateMany/etc...

### ISSUE THREAD

**Issue by mramato** [reduxjs/redux-toolkit#1064]
I stumbled on #532 while trying to determine if calling `updateOne` with an id that may not be in the collection is a "guaranteed safe" no-op rather than just a quirk of the current implementation. It is not mentioned in the https://redux-toolkit.js.org/api/createEntityAdapter#crud-functions sectinon of the documentation.  Am I correct in that it's a guarantee? If so, would you consider adding that fact to the above doc link?

**Comment by markerikson** [comment]
Yeah, that's intentional behavior:

https://github.com/reduxjs/redux-toolkit/blob/ed125f56dd18b3dd02ea7fa709fb7a498610c947/src/entities/unsorted_state_adapter.ts#L131-L132

Please feel free to file a PR that tweaks the wording in that docs page.

**Comment by mramato** [comment]
Yep, that's me. We were using redux for a few months before I noticed that you were a redux maintainer as well as a CesiumJS user (at least at the time, sorry to hear you aren't on that project any more).

We're using it for both our [Smart Construction](https://cesium.com/industries/digital-transformation/smart-construction/) work and with [Cesium Stories](https://cesium.com/platform/cesium-ion/cesium-stories/), though the latter is on old school redux and really needs to be migrated to RTK.

I'll try to open a PR to tweak the doc later this week, thanks!

**Comment by akashshyamdev** [comment]
@markerikson can I handle this?
